r_scripts/data_wrangle/oregon_casecrossover.R

```diff
@@ -0,0 +1,106 @@
+# ------------------------------------------------------------------------------

```

Comment:
Jingyang, If this takes awhile, let's consider running it in parallel. Should be relatively easy to modify.